cypress/support/step-definitions/menu-steps.js

```diff
@@ -78,3 +81,40 @@ Then("{string} submenu has alternate colour theme", (position) => {
     "rgb(230, 235, 237)"
   );
 });
+
+When("I hover over default menu fifth expandable Menu component", () => {
+  onClickMenuItem().trigger("mouseover");
+});
+
+Then("Menu fifth expandable element submenu is not visible", () => {
+  submenuItem().should("not.exist");
+});
+
+When("I click default menu fifth expandable Menu component", () => {
+  onClickMenuItem().click();
+});
+
+Given(
+  "I Tab {int} times to default menu fifth expandable Menu component",
+  (times) => {
+    noActionMenuItem().click();
+    cy.focused().trigger("keydown", { keyCode: 27, which: 27 });
```

Comment:
I think we could here a `keyCode()` from `helper.js` file instead of hardcoding the `27` value here